pkg/upgrade/upgrader/jiva_upgrade.go

```diff
@@ -429,6 +469,18 @@ func (j *jivaVolumeOptions) replicaUpgrade(openebsNamespace string) error {
 	}
 
 	statusObj.Phase = utask.StepErrored
+
+	err = scaleDownTargetDeploy(j.controllerObj.name, j.ns)
```

Comment:
is `j.controllerObj` be NULL here? it seems to be populated in next step

Reply:
It is populated before the patching is done in the preuprade step. https://github.com/openebs/maya/blob/4104241e9966c87bc3153d7dab9aedaba9f503ac/pkg/upgrade/upgrader/jiva_upgrade.go#L599